tests/server.test.js

```diff
@@ -0,0 +1,52 @@
+const { MongoMemoryServer } = require('mongodb-memory-server');
```

Comment:
@IgorChaikin Lets use test files structure, which matches with your routes structures. For example

```
server/routers/admin.js
server/routers/admin.test.js
server/routers/api.js
server/routers/api.test.js
server/routers/auth.js
server/routers/auth.test.js
```

Reply:
@IgorChaikin Lets put `*.test` files near files, which are tested, you can move `tests/cookiesParser.js` to `utils` or `lib` directory